### ISSUE TITLE
Pre-training with a single real image and shape augmentation.

### ISSUE THREAD

**Issue by cugwu** [SUPER-TADORY/1p-frac#2]
Can the parameters of the affine transform used to create the dataset with a single real image be known? Thanks in advance for the answer

**Comment by qian39** [comment]
Hi,

I've been fine-tuning the pre-trained model vit_tiny_patch16_224_sigma3.5_delta0.1_sample1000_80000ep on CIFAR-100, but I'm only achieving around 75% accuracy instead of the expected 84.2%. I'm using a batch size of 768 and a GPU with the following setup: NVIDIA RTX A6000, Driver Version: 535.183.01, CUDA Version: 12.2.

Do you have any suggestions on how to improve the accuracy?

Thank you!
